dotCMS/src/main/java/com/dotcms/contenttype/business/init/DotAnnouncementsInitializer.java

```diff
@@ -0,0 +1,82 @@
+package com.dotcms.contenttype.business.init;
+
+import com.dotcms.config.DotInitializer;
+import com.dotcms.contenttype.business.ContentTypeAPI;
+import com.dotcms.contenttype.model.field.Field;
+import com.dotcms.contenttype.model.field.ImmutableDateField;
+import com.dotcms.contenttype.model.field.ImmutableSelectField;
+import com.dotcms.contenttype.model.field.ImmutableTextField;
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.contenttype.model.type.ImmutableSimpleContentType;
+import com.dotcms.contenttype.model.type.SimpleContentType;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.folders.model.Folder;
+import com.dotmarketing.portlets.workflows.business.WorkflowAPI;
+import com.dotmarketing.util.Logger;
+import io.vavr.control.Try;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class DotAnnouncementsInitializer implements DotInitializer {
```

Comment:
Announcements is not a content type in every dotCMS?  It is a single content type on https://www.dotcms.com?

What is this code for?

Reply:
code removed